src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -4306,6 +4306,62 @@ public void XA1018 ()
 			}
 		}
 
+		[Test]
+		public void XA1024XA1025 ([Values (null, "proguard", "r8")] string linkTool, [Values (null, "true", "false")] string enableProguard, [Values (null, "true", "false")] string androidEnableProguard)
```

Comment:
Since this is adding 9 tests (3 * 3 * 3)... can we use `[TestCaseSource]` and list a few less combinations?

Maybe we could get by with 3-5 combinations or something?